qemu/tests/cfg/virtio_port_hotplug.cfg

```diff
@@ -37,8 +37,13 @@
             unplug_chardev_vs1 = no
             unplug_chardev_vs2 = no
             filesize = 100
-            Linux:
-                filesize = 2000
+            variants:
+                - @default:
+                    bg_test = no
```

Comment:
Pls keep them alignment with line37 in py file.
yes/no 
true/false

Reply:
@xiagao  They are not same meaning, you can check the function `get_boolean`.

Reply:
Yes, just now missed get_boolean function.

Reply:
> Pls keep them alignment with line37 in py file.
> yes/no
> true/false

please check the `get_boolean(self, key, default=False)`,  'yes/no' is the `key`, True/False is the default `boolean` and we need to set the `key` value.